scripts/validate-release

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+set -e
+
+BUILD_REGEX="build[0-9]+"
+
+info() {
```

Comment:
At some point we're gonna need a shell package to source these types of things in since they're used in some many placed. No action. Just sayin... :)